components/user/UserProfile.js

```diff
@@ -68,9 +67,9 @@ function UserProfile({ BASE_URL, data }) {
           </div>
         </div>
       </div>
-      <div className="flex justify-center my-4 text-center">
+      <div className="justify-center my-4 text-center whitespace-pre-wrap">
         <ReactMarkdown components={{ a: LinkRenderer }}>
-          {data.bio}
+          {data.bio.replace(/\n/gi, "\n &nbsp;")}
```

Comment:
Should this happen in the markdown plugin, so all output from text area can benefit from multiple lines?

Reply:
This worked well for the Profile...

<img width="1328" alt="Screenshot 2023-07-23 at 15 14 58" src="https://github.com/EddieHubCommunity/LinkFree/assets/624760/e0e7e5c9-3318-4ef3-95b0-e526177d79c4">

But other textareas will need this functionality also, for example events has a textarea but the output does not obey the same line breaks ...

<img width="1295" alt="Screenshot 2023-07-23 at 15 15 39" src="https://github.com/EddieHubCommunity/LinkFree/assets/624760/c08d0ce4-c903-474b-8241-c67366bd93c3">

Reply:
> Should this happen in the markdown plugin, so all output from text area can benefit from multiple lines?

Will you please help me in this? how can I make changes in plugin code to make the things work